sections/heroes.tex

```diff
@@ -12,21 +12,21 @@ \subsection*{Main Hero}
 Each Faction's Main Hero has 3 Movement Points.
 Only the Main Hero can use Your Deck.\par
 Each Main Hero starts the game at Level 1 and can advance up to Level 7 by gaining Experience.
-Experience is gained from \hyperlink{Combatexperience}{winning Combat}, Visiting certain \hyperlink{All Map Locations}{Locations} and the \hyperlink{Treasure Die}{Treasure Die} \includesvg[height=10px]{\svgs/treasure.svg}.
+Experience is gained from \pagelink{Combatexperience}{winning Combat}, Visiting certain \pagelink{All Map Locations}{Locations} and the \pagelink{Treasure Die}{Treasure Die} \includesvg[height=10px]{\svgs/treasure.svg}.
```

Comment:
```suggestion
Experience is gained from \pagelink{Combatexperience}{winning Combat}, Visiting certain \pagelink{All Map Locations}{Locations} and the \pagelink{Treasure Die}{Treasure Die \includesvg[height=10px]{\svgs/treasure.svg}}.
```
I think it's better to make the glyph a part of the link, so that it's not rendered behind the page number.

![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/d1d13d30-7bd7-4a7d-8e30-c1833d6e325d)